amp.php

```diff
@@ -177,6 +177,18 @@ function amp_init() {
 
 	// Add actions for legacy post templates.
 	add_action( 'wp', 'amp_maybe_add_actions' );
+
+	// Broadcast plugin updates.
+	$old_version = AMP_Options_Manager::get_option( 'version', '0.0' );
+	if ( version_compare( $old_version, AMP__VERSION ) ) {
+		/**
+		 * Triggers when after amp_init when the plugin version has updated.
```

Comment:
This is a very minor point, but maybe `Triggers when after amp_init` should be `Triggers after amp_init`